src/popup/components/InputAmountV2.vue

```diff
@@ -101,12 +101,15 @@ export default defineComponent({
 
     const currentAsset = computed((): IAsset => props.selectedAsset || aeToken);
     const hasError = computed(() => (root as any).$validator.errors.has('amount'));
-    const totalAmount = computed(
-      () => (currentAsset.value?.current_price)
-        ? ((+props.value || 0) * currentAsset.value.current_price).toFixed(2)
+    const totalAmountFormatted = computed(() => formatCurrency(
+      (currentAsset.value?.current_price)
```

Comment:
```suggestion
      currentAsset.value?.current_price
```

Reply:
I already mentioned few times that wrapping the condition with braces in one liners makes the code more readable. At least in my opinion.